app/views/layouts/base.html.slim

```diff
@@ -13,6 +13,8 @@ html lang="en-US"
     
     / TODO: Find a better location for this (only used in loan criteria charts)
     = javascript_include_tag "https://www.gstatic.com/charts/loader.js"
+    javascript:
+      google.charts.load('current', {'packages':['corechart']});
```

Comment:
One of the requested fixes in this branch is: "Figure out styling of percentages as some numbers are uncentered in some cases and cut off, in other cases."

The tree that the questions are loaded into is interfering with the usual rendering of Google Charts. A chart can be loaded fine outside of our questionnaire tree and the percentages hide and show based on how large the pie slice is, as expected. Inside the tree, the visual representation of the chart is skewed and the interactions with the chart are disabled or not working. In print view, one can click on a pie slice and see more information about the slice (percent and amount). Though I'm not sure we want that functionality inside the print view, it does make sense inside the loan view where one can edit and view the questionnaire. But clicking the chart does not work inside that context.